static/components/customer-stall/customer-stall.html

```diff
@@ -1,79 +1,35 @@
 <div>
   <q-toolbar>
     <q-breadcrumbs class="cursor">
-      <q-breadcrumbs-el
-        label="Market"
-        icon="home"
-        @click="$emit('change-page', 'market')"
-        style="cursor: pointer"
-      ></q-breadcrumbs-el>
-      <q-breadcrumbs-el
-        :label="stall?.name || 'Stall'"
-        icon="widgets"
-      ></q-breadcrumbs-el>
+      <q-breadcrumbs-el label="Market" icon="home" @click="$emit('change-page', 'market')"
```

Comment:
- shouldn't the label dynamic be like in `customer-market`?
`<q-breadcrumbs-el :label="styles.name ?? 'Market'"`

Reply:
Yes...